pkgs/state-transition/src/lib.zig

```diff
@@ -84,14 +83,14 @@ test "mock genesis and block production" {
 
     // check produced block 1 state root
     var expected_block1_state_root: [32]u8 = undefined;
-    _ = try std.fmt.hexToBytes(expected_block1_state_root[0..], "f77aaa703c400ccaffa8e674316713b044fcc3d94ec5764b00ce7edc138e7c95");
+    _ = try std.fmt.hexToBytes(expected_block1_state_root[0..], "993a459ace700aaeacf2fd3ec20b0f9efd3acae93ff904c5adbc1997b025ea97");
```

Comment:
if you hardcode stuff, presumably because you got it from the lodestar ssz library, it'd be nice to add a comment to let people know how to reproduce that.